system_tests/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM google/python
+
+# Inject secrets
+# TODO: Encrypt/decrypt secrets
+ADD data/ data/
+ENV GOOGLE_APPLICATION_CREDENTIALS /home/vmagent/app/data/cloud-python-runtime-qa-credentials.json
+
+# Get the source.
+RUN git clone https://github.com/GoogleCloudPlatform/google-cloud-python.git
+WORKDIR google-cloud-python
+
+# Checkout the latest release.
+RUN git checkout $(git describe --tags --abbrev 0)
+
+# Install gcloud so we can do test setup
+RUN apt-get update && apt-get install curl
+RUN curl https://sdk.cloud.google.com | bash
```

Comment:
Filed #40 and https://github.com/GoogleCloudPlatform/debian-docker/issues/50 .

Meanwhile, this is good enough for me.